geoparquet/geoparquet-example.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
> which we really want to avoid

I agree, but I think the GeoParquet story is a little young for this, unless we describe how to use the underlying `pyarrow` library and do the conversion to GeoParquet manually. There's not really cloud-native support in GeoPandas yet. Even the examples at the bottom of the notebook are reading the _entire file_, so unless you're running in AWS the example will be slow (e.g. it took 6 minutes to read the file for me)